### PR TITLE
fix/LWAT-26335403: AElf ledger wallet README file updates

### PR DESCRIPTION
**Pull request by Faizal-aelf**
**AElf ledger wallet README file updates.** 

- Replaced "Companion Wallet" text with "AElf Ledger Wallet".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Ledger Companion Wallet (AElf version)
+# AElf Ledger Wallet
```
Comment (yongenaelf) [inline README.md]: AELF or aelf

File: README.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ You may also see any lint errors in the console.
 
 - [Ledger Web USB/HID walkthrough](https://developers.ledger.com/docs/transport/web-hid-usb/)
 - [Ledger Web Bluetooth walkthrough](https://developers.ledger.com/docs/transport/web-bluetooth/)
-- [How to use your Ledger Device with the AElf Companion Wallet](https://docs.google.com/document/d/1xxl_DRUNoqfWF__nO_rhUBPDA6-yNf0Y3nfTie0OQwA/edit?usp=sharing)
+- [How to use your Ledger Device with the AElf Ledger Wallet](https://docs.google.com/document/d/1xxl_DRUNoqfWF__nO_rhUBPDA6-yNf0Y3nfTie0OQwA/edit?usp=sharing)
```
Comment (yongenaelf) [inline README.md]: AELF or aelf

**Review by yongenaelf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Faizal-aelf) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (Faizal-aelf) [reply README.md]: Updated

**Review by yongenaelf — APPROVED** [review]

**Pull request merged**